python/lsst/ts/standardscripts/auxtel/startup.py

```diff
@@ -0,0 +1,78 @@
+# This file is part of ts_externalcripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["Startup"]
+
+import yaml
+
+from lsst.ts import salobj
+from lsst.ts.observatory.control.auxtel.atcs import ATCS, ATCSUsages
+
+
+class Startup(salobj.BaseScript):
```

Comment:
Please consider a longer, more descriptive name. (Possibly both here and in ts_observatory_control.

I confess to some surprise that the script (and method in ts_observatory_control) has no configuration. Naively I would have expected you might want to do "everything but open the slit", for example, if you want to exercise the telescope while it is raining or the humidity is too high.